apps/portals/standards/src/pages/StandardsDetailsPage.tsx

```diff
@@ -28,7 +42,58 @@ export const standardDetailsPageContent: DetailsPageContentType = [
     id: 'About The Standard',
     title: 'About The Standard',
     element: (
-      <MarkdownSynapseFromColumnData columnName={'studyDescriptionLocation'} />
+      <DetailsPageContextConsumer columnName={'Name'}>
+        {({ value, context }) => {
+          const query: Query = {
+            sql: dataSql,
+            additionalFilters: getAdditionalFilters(
+              undefined,
+              { Name: value! },
+              ColumnSingleValueFilterOperator.EQUAL,
+            ),
+          }
+          if (context.rowData && context.rowSet) {
+            return (
+              <RowDataTable
+                rowData={
+                  context.rowData.values?.filter(
+                    (value): value is string => value !== null,
+                  ) ?? []
+                }
+                headers={context.rowSet?.headers ?? []}
+                labels={[
+                  'Name',
+                  'Collections',
+                  'Organizations',
+                  'Data_Topic',
+                  'Acronym',
+                  'Is_Open',
+                  'Registration',
+                ]}
+                columnAliases={dataColumnAliases}
+              />
+            )
+          } else {
+            return (
+              <RowDataTableWithQuery
+                row={context.rowData ?? { values: [] }}
+                headers={context.rowSet?.headers ?? []}
+                query={query}
+                labels={[
+                  'Name',
+                  'Collections',
+                  'Organizations',
+                  'Data_Topic',
+                  'Acronym',
+                  'Is_Open',
+                  'Registration',
+                ]}
+                columnAliases={dataColumnAliases}
+              />
+            )
```

Comment:
We should always (eventually) have `rowData` and `rowSet`, so in this case we should show a loading component like a skeleton.